zipkin-server/src/main/java/zipkin/server/ZipkinQueryApiV1.java

```diff
@@ -43,6 +44,7 @@
  */
 @RestController
 @RequestMapping("/api/v1")
+@CrossOrigin("${zipkin.query.allowed-origins:'*'}")
```

Comment:
Yeah the single quotes aren't needed - removed them. 

Happy to try and get some ITs in there - are the ones in `zipkin-server/src/it` a good example to start with?